### DISCUSSION TITLE
JSONSchema Support for Kafka

### DISCUSSION THREAD

**Discussion by xdgrulez** [trinodb/trino#22420] [Ideas]
Hi,

we would like to use Trino for our new Kafka-based data integration strategy here at Migros.

However, currently, Trino only supports Avro and Protobuf, but not JSONSchema - and we use JSONSchema at lot.

Is there any chance anybody has already worked on Kafka/JSONschema support for Trino?

Best, Ralph

**Comment by hashhar** [comment]
No, there is no JSONSchema integration with the Confluent schema registry implemented. Someone would need to work on this.

**Comment by xdgrulez** [comment]
Thanks :)

Am Di., 25. Juni 2024 um 12:28 Uhr schrieb Ashhar Hasan <
***@***.***>:
